sass/components/_popover.scss

```diff
@@ -301,7 +307,7 @@
 .user-popover__role {
     padding: 3px 5px;
     margin-left: 10px;
-    background-color: $gray;
+    background: rgba(var(--center-channel-color-rgb), 0.3);
```

Comment:
Probably out of the scope of this PR, but would it be possible to create some custom lint rule that takes care of enforcing these.

Reply:
well ... that would be helpful ... and there probably is ... at least for eslint you can define a regex rule that checks for stuff like this

Reply:
@hmhealey Should we create a ticket for web platform? Or what do you think about this?

Reply:
I just found a stylelint rule that could enforce that by setting specific properties and a regex for the value:

https://github.com/AndyOGo/stylelint-declaration-strict-value

Reply:
@larkox follow-up ticket here: [MM-40158](https://mattermost.atlassian.net/browse/MM-40158)